### ISSUE TITLE
adapter won't start after update 2.0.11 -> 2.0.12

### ISSUE THREAD

**Issue by tetzlav** [ioBroker/ioBroker.knx#237]
```
knx.0 | 2022-03-01 10:14:26.627 | warn | Terminated (UNCAUGHT_EXCEPTION): Without reason
-- | -- | -- | --
knx.0 | 2022-03-01 10:14:26.627 | info | terminating
knx.0 | 2022-03-01 10:14:26.480 | error | val is not defined
knx.0 | 2022-03-01 10:14:26.480 | error | ReferenceError: val is not defined     at _0x133b3c.<anonymous> (/opt/iobroker/node_modules/iobroker.knx/knx.js:1:357258)     at _0x133b3c.emit (events.js:400:28)     at _0x133b3c.emit (domain.js:475:12)     at Socket.<anonymous> (/opt/iobroker/node_modules/iobroker.knx/knx.js:1:104254)     at Socket.emit (events.js:400:28)     at Socket.emit (domain.js:475:12)     at UDP.onMessage (dgram.js:931:8)
knx.0 | 2022-03-01 10:14:26.479 | error | uncaught exception: val is not defined
knx.0 | 2022-03-01 10:14:26.330 | info | Connecting to knx GW: 192.168.178.59:3671 with phy. Adr: 1.1.101

**Comment by chefkoch009** [comment]
is fixed in next release.

**Comment by chefkoch009** [comment]
is fixed in v2.0.13. Otherwise please re-open this issue.

**Comment by tetzlav** [comment]
v2.0.13 works fine again
